dm/config/security/security.go

```diff
@@ -95,5 +97,23 @@ func (s *Security) Clone() *Security {
 	clone.SSLCABytes = append([]byte(nil), s.SSLCABytes...)
 	clone.SSLKeyBytes = append([]byte(nil), s.SSLKeyBytes...)
 	clone.SSLCertBytes = append([]byte(nil), s.SSLCertBytes...)
+	clone.SSLCA = s.SSLCA
+	clone.SSLCert = s.SSLCert
+	clone.SSLKey = s.SSLKey
 	return &clone
 }
+
+// WriteTLSContentToFiles write tls content to temp file and update tls path fields.
+func (s *Security) WriteTLSContentToFiles(fileName string) error {
```

Comment:
when and why write to temp file?

Reply:
when write to temp file need meet few condition:
1.  when we call `GetLightningConfig()`
2. subtask config set `LoaderConfig.Security` (now only openapi can set the field)
3. `LoaderConfig.Security` only set certificates content and do not set certificates path.

> why write to temp file
Since only set certificates content cannot verify certificates correctly. Yes, just like your comment below.